db2/animaldb_table.md

```diff
@@ -0,0 +1,8 @@
+# Create a SQL table
+
+1. Create a Db2 table to house the data.
+    1. `Manage`>`Open` to get the Db2 Warehouse on Cloud dashboard, then choose `Run SQL`.
+    1. Create a table called `ANIMALS` with appropriate columns for the data by pasting this SQL statement:
```

Comment:
Add `<br/>` to have the CREATE TABLE start on it's own line.

Reply:
Fixed in 3f20428159f2d4f02923221d5b76ae89ff96b98d